server/constants/timeslot-status.json

```diff
@@ -0,0 +1,3 @@
+{
+    "available": "available",
+    "unavailable": "unavailable"}
```

Comment:
Minor coding best practice: I would recommend moving the final curly braces to the next line to comply with typical json file structures.

Reply:
Thanks Amy!